RELEASE.md

```diff
@@ -4,15 +4,15 @@
 
 ###v0.9.6
 
-November, 2015 | [Release Notes](https://github.com/PredictionIO/PredictionIO/blob/master/RELEASE.md) have been moved to Github and you are reading them. For a detailed list of commits check [this page](https://github.com/PredictionIO/PredictionIO/commits/master)
+November, 2015 | [Release Notes](https://github.com/apache/incubator-predictionio/blob/develop/RELEASE.md) have been moved to Github and you are reading them. For a detailed list of commits check [this page](https://github.com/apache/incubator-predictionio/commits/develop)
```

Comment:
@HyukjinKwon Release notes should refer to stable sources, so we should use `master`'s `RELEASE.md`.